prov/efa/src/rdm/rxr_srx.c

```diff
@@ -0,0 +1,188 @@
+ */
+static int rxr_srx_queue_msg(struct fi_peer_rx_entry *rx_entry)
+{
+    return -FI_ENOSYS;
+}
+
+/**
+ * @brief This call is invoked by the peer provider to queue
+ * a received tagged message to owner provider's unexpected queue,
+ * when the peer calls owners->get_msg() but the owner fails to
+ * find a matching receive buffer.
+ *
+ * @param[in] rx_entry the entry to be queued
+ * @return int 0 on success, a negative integer on failure
+ */
+static int rxr_srx_queue_tag(struct fi_peer_rx_entry *rx_entry)
+{
+	return -FI_ENOSYS;
+}
+
+/**
+ * @brief This call is invoked by the peer provider to release a rx entry
+ * from owner provider's resource pool
+ *
+ * @param[in] entry the entry to be freed
+ */
+static void rxr_srx_free_entry(struct fi_peer_rx_entry *entry)
+{
+}
+
+/**
+ * @brief This call is invoked by the owner provider to start progressing
+ * the rx_entry that matches a received message.
+ *
+ * @param rx_entry the rx entry to be progressed.
+ * @return int 0 on success, a negative integer on failure.
+ */
+static int rxr_srx_start_msg(struct fi_peer_rx_entry *rx_entry)
+{
+    return -FI_ENOSYS;
+}
+
+/**
+ * @brief This call is invoked by the owner provider to start progressing
+ * the rx_entry that matches a received tagged message.
+ *
+ * @param rx_entry the fi_peer_rx_entry to be progressed.
+ * @return int 0 on success, a negative integer on failure.
+ */
+static int rxr_srx_start_tag(struct fi_peer_rx_entry *rx_entry)
+{
+    return -FI_ENOSYS;
+}
+
+/**
+ * @brief This call is invoked by the owner provider to discard
+ * the message and data associated with the specified fi_peer_rx_entry.
+ * This often indicates that the application has canceled or discarded
+ * the receive operation.
+ *
+ * @param rx_entry the fi_peer_rx_entry to be discarded.
+ * @return int 0 on success, a negative integer on failure.
+ */
+static int rxr_srx_discard_msg(struct fi_peer_rx_entry *rx_entry)
+{
+    return -FI_ENOSYS;
+}
+
+/**
+ * @brief This call is invoked by the owner provider to discard
+ * the tagged message and data associated with the specified fi_peer_rx_entry.
+ * This often indicates that the application has canceled or discarded
+ * the receive operation.
+ *
+ * @param rx_entry the fi_peer_rx_entry to be discarded.
+ * @return int 0 on success, a negative integer on failure.
+ */
+static int rxr_srx_discard_tag(struct fi_peer_rx_entry *rx_entry)
+{
+    return -FI_ENOSYS;
+}
+
+
+static struct fi_ops_srx_owner rxr_srx_owner_ops = {
+	.size = sizeof(struct fi_ops_srx_owner),
+	.get_msg = rxr_srx_get_msg,
+	.get_tag = rxr_srx_get_tag,
+	.queue_msg = rxr_srx_queue_msg,
+	.queue_tag = rxr_srx_queue_tag,
+	.free_entry = rxr_srx_free_entry,
+};
+
+static struct fi_ops_srx_peer rxr_srx_peer_ops = {
+	.size = sizeof(struct fi_ops_srx_peer),
+	.start_msg = rxr_srx_start_msg,
+	.start_tag = rxr_srx_start_tag,
+	.discard_msg = rxr_srx_discard_msg,
+	.discard_tag = rxr_srx_discard_tag,
+};
```

Comment:
Suggest moving to L106 to denote the end of peer ops

Reply:
I guess you are saying moving `rxr_srx_owner_ops` to L106. I prefer to keep it in this way so the ops function table assignments can be put together.